pkg/sql/table.go

```diff
@@ -360,7 +362,7 @@ func (tc *TableCollection) getTableVersionByID(
 		}
 		return nil, err
 	}
-	tc.timestamp = txn.OrigTimestamp()
+	tc.timestamp = origTimestamp
```

Comment:
We need to add a comment to both both calls made to txn.UpdateDeadlineMaybe() in this file.

// The descriptors picked from the lease manager are picked using the OrigTimestamp. The actually commit timestamp can be different so set the deadline for the transaction.